cartographer/mapping/proto/submap.proto

```diff
@@ -16,15 +16,21 @@ syntax = "proto2";
 
 package cartographer.mapping.proto;
 
-import "cartographer/transform/proto/transform.proto";
 import "cartographer/mapping_2d/proto/probability_grid.proto";
 import "cartographer/mapping_3d/proto/hybrid_grid.proto";
+import "cartographer/transform/proto/transform.proto";
 
-message Submap {
+message Submap2D {
   optional transform.proto.Rigid3d local_pose = 1;
   optional int32 num_range_data = 2;
   optional bool finished = 3;
```

Comment:
Maybe add a comment here? It's not clear to me what this should mean. Clearly it's "finished" since it's been serialized...